components/login/passwordReset/README.md

```diff
@@ -0,0 +1,36 @@
+# LoginPasswordReset
+
+Work in progress.
```

Comment:
🍆 something we could already add at this point, right?

Reply:
We'll add ASAP 😅 
As we'll be working on this on the following days, a lot of Prs will be submitted to add features and complete the component. This is just a fast scaffolding to ensure we share the same component structure.